cinder/README.md

```diff
@@ -71,18 +71,19 @@ cp /usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-vnx-c
 
 Modify backend configuration in `cinder-dellemc-vnx-config.yaml`.
 
+Note: **LVM driver** is enabled by default in TripleO, you want to set the ```CinderEnableIscsiBackend``` to false in one of your environment file to turn it off.
+```yaml
```

Comment:
It's better to add a blank line before and after the code block, to keep the format be consistent with other code blocks.

Reply:
> It's better to add a blank line before and after the code block, to keep the format be consistent with other code blocks.

Done